UnityProject/Assets/Scripts/Disposals/DisposalPipe.cs

```diff
@@ -7,6 +7,8 @@ namespace Disposals
 	[CreateAssetMenu(fileName = "DisposalPipe", menuName = "Tiles/Disposal Pipe", order = 1)]
 	public class DisposalPipe : BasicTile, IExaminable
 	{
+// Ignore never-assigned-to warning as it is assigned via inspector
+#pragma warning disable CS0649
 		[Tooltip("Set the type of disposal pipe this is.")]
```

Comment:
Dont need to do the warning things here either. For lists do = new List<type>();